GOVERNANCE.md

```diff
@@ -0,0 +1,118 @@
+
+A Call for Agreement is mandatory when:
+* A Chair determines that the topic requires a Call for Agreement.
+* When petitioned by members of the Operator Framework Community and submitted to the Chairs to call a vote unless rejected for cause.
+* Technical decisions that add, remove, or change dependencies and requirements.
+* Revoke a previous decision made by the Call for Agreement process.
+
+Once the Call for Agreement voting period has elapsed, all votes are counted, with at least a 51% majority of votes needed for consensus. A Chair will then declare the agreement “accepted” or “declined”, ending the Call for Agreement.
+
+Once rejected, a Call for Agreement must be revised before re-submission for a subsequent vote. All rejected Calls for Agreement will be reported to the Operator Framework Community as rejected.
+
+### Charter and Governing Documents
+
+The Operator Framework Community may, from time to time, adopt or amend its Governing Documents and Charter, using a modified Call for Agreement process:
+* A quorum of at least 51% of active Operator Framework Community Members must vote.
+* A quorum of 3 Chairs is needed
+* Two-thirds of the voting quorum must approve the proposal.
+* A majority of Chairs must approve the proposal.
+* A public notice period of no less than 14 days from the Call for Agreement proposing the change must elapse before voting begins.
+* Any Call for Agreement that follows this process is considered a Governing Document. 
+
+For initial approval of this Charter via Call for Agreement all members are eligible to vote, even those that have been a member for less than 21 days. This Charter will be approved if there is a majority of positive votes.
+
+## Organizational Roles
+
+### Role of Chairs
+
+The primary role of Chairs is to run operations and the governance of the group. The Chairs are responsible for:
+* Setting the agenda for meetings.
+* Extending discussion via asynchronous communication to be inclusive of members who cannot attend a specific meeting time.
+* Scheduling discussing of proposals that have been submitted.
+* Asking for new proposals to be made to address an identified need.
+* Oversee disciplinary action for members. The Chairs have the authority to declare a member inactive and expel members for cause.
+* Chairs will serve for one-year revolving terms and will be approved using the [Condorcet Method](https://en.wikipedia.org/wiki/Condorcet_method). Upon the expiration of a Chair’s term, it will continue for another year, unless challenged.
+
+The terms for founding Chairs start on the approval of this charter.
+
+When no candidate has submitted their name for consideration, the current Chairs may appoint an acting Chair until a candidate comes forward.
+
+Chairs must be active members. Any inactivity, disability, or ineligibility results in immediate removal.
```

Comment:
is `disability` the word we want to use here?

Reply:
agreed, `inability` feels better, or `unable to fulfill the role` to be even more gentle. Also, this immediate removal seems fairly aggressive. How does this removal happen? By simple majority of the existing chairs? I'd prefer to identify ways that existing chairs can gracefully terminate their role and trigger an election, and provide mechanisms that reflect reality. This could potentially be an enhancement proposal after the initial governance charter has landed. :)